homeassistant/components/homeassistant_hardware/silabs_multiprotocol_addon.py

```diff
@@ -319,7 +401,8 @@ async def async_step_on_supervisor(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
         """Handle logic when on Supervisor host."""
-        addon_info = await self._async_get_addon_info()
+        addon_manager: AddonManager = await get_addon_manager(self.hass)
+        addon_info = await self._async_get_addon_info(addon_manager)
```

Comment:
Since we're juggling two add-ons and the code is growing I think we should name variables and functions - including the flow steps - by which add-on they interact with (`multipan_addon_manager = .. `) etc.